facdb_build/sql/dcp_colp.sql

```diff
@@ -52,7 +54,10 @@ SET hash =  md5(CAST((t.*)AS text)),
 				WHEN parcelname <> ' ' AND parcelname IS NOT NULL THEN initcap(parcelname)
 				ELSE initcap(REPLACE(usetype, 'OTHER ', ''))
 			END),
-	factype = initcap(REPLACE(usetype, 'OTHER ', '')),
+	factype = (CASE
+				WHEN parcelname ~* 'PRECINCT' AND usecode = '0500' THEN 'POLICE STATION'
```

Comment:
Should this be `WHEN parcelname ~* 'PRECINCT' AND usecode = '0500' THEN 'Police Station'`, since the other case uses `initcap`?

Reply:
All the factypes in the facilities table are in upper case.  I think we're switching it back to upper case at the end.  Yes, for consistency I'll make it 'Police Station'